rmw_zenoh_cpp/src/detail/event.hpp

```diff
@@ -60,7 +60,7 @@ struct rmw_zenoh_event_status_t
   size_t total_count;
   size_t total_count_change;
   size_t current_count;
-  int32_t current_count_change;
+  int64_t current_count_change;
```

Comment:
We still want to keep this as `int32_t` and apply the clamps within update_event_status.